roxctl/common/flags/endpoint_test.go

```diff
@@ -0,0 +1,85 @@
+package flags
+
+import (
+	"testing"
+
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestEndpointAndPlaintextSetting(t *testing.T) {
+	truth := false
+	endpointChanged = &truth
+	plaintextSet = &truth
+
+	testCases := []struct {
+		endpoint     string
+		host         string
```

Comment:
I'd suggest to make it clear here what are the inputs and what are the outputs. Maybe describe the setup of the preconditions (e.g., env var is not set, endpoint flag value is passed as input, etc) and update fields' names to, e.g., `endpointFlagValue` and `expectedEndpoint`.

Reply:
given/expectedEndpoint will do the work